torch/_dynamo/trace_rules.py

```diff
@@ -3327,7 +3327,11 @@ def _module_dir(m: types.ModuleType):
     "torch._export",
     "torch._functorch",
     "torch._guards",
-    "torch._higher_order_op",
+    "torch._higher_order_ops._invoke_quant",
+    "torch._higher_order_ops.effects",
+    "torch._higher_order_ops.map",
+    "torch._higher_order_ops.torchbind",
+    "torch._higher_order_ops.wrap",
```

Comment:
Some of these have buggy code that I'll resolve in the future. For example, I think the code for MapWrapper (in map.py) is wrong